app/page.tsx

```diff
@@ -1,81 +1,153 @@
 import Image from 'next/image'
-import {Auth} from '~/components/Auth'
-import {Footer} from '~/components/Footer'
+import Link from 'next/link'
+import {env} from 'process'
+import future from '~/assets/future.png'
+import joshuaTreeDay from '~/assets/joshua-tree-day.png'
+import joshuaTreeNight from '~/assets/joshua-tree-night.png'
 import {Header} from '~/components/Header'
-import Cal from '~/components/logos/Cal'
-import Highlight from '~/components/logos/Highlight'
-import Precedent from '~/components/logos/Precedent'
-import Trigger from '~/components/logos/Trigger'
-import demoSrc from '../public/assets/demo.png'
+import {Cal, Highlight, Precedent, Trigger} from '~/components/logos'
+import {Nuxt} from '~/components/logos/Nuxt'
+
+const loomConfig = {
+	hide_owner: 'true',
+	hideEmbedTopBar: 'true',
+	hide_share: 'true',
+	hide_title: 'true',
+	default_speed: 'true',
+	skip_embed_eovn: 'true'
+}
+const loomQueryParams = new URLSearchParams(loomConfig).toString()
+
+const CTAButton = () => (
```

Comment:
Heads up, this button should sign you in now instead of redirecting to the app's GitHub page. [Check here](https://github.com/RubricLab/maige/blob/main/lib/components/Auth.tsx).